test/test_inference/test_imports.py

```diff
@@ -307,6 +307,25 @@ def import_names(*args, **kwargs):
     assert 'path' in import_names(s, column=len(s) - 3)
 
 
+def test_duplicated_import(Script):
+    def import_names(*args, **kwargs):
+        return [d.name for d in Script(*args).complete(**kwargs)]
+
+    s = 'import os, o'
+    assert 'os' not in import_names(s)
+    assert 'os' in import_names(s, column=len(s) - 3)
+
+    s = 'from os import path, p'
+    assert 'path' not in import_names(s)
+    assert 'path' in import_names(s, column=len(s) - 3)
+
+    s = 'import path as pp, p'
+    assert 'path' not in import_names(s)
+
+    s = 'from os import path as pp, p'
+    assert 'path' not in import_names(s)
```

Comment:
Can you also add a test here for 
```
s = 'from os import chdir as path, p'
assert 'path' in import_names(s)
```

I feel like this is still wrong.

Reply:
Yes, it will be wrong. I mentioned it above:
> import alias will also be filtered Generally, there is unlikely to be a situation where an alias is named exactly the same as a specific original name.

If you think it unacceptable，I can parse this syntax and remove it from the filter

Reply:
Oh sorry, I didn't read that.

> If you think it unacceptable，I can parse this syntax and remove it from the filter

I think it is acceptable, but it is the kind of thing I always fix before merging. So if you don't do it, I have to :)

I think it's also a very easy fix (probably one simple if).

Reply:
Also know that this piece of software is used by millions of people, so issues like this one will appear sooner or later.

Reply:
Done～